zephyr/CMakeLists.txt

```diff
@@ -803,8 +803,8 @@ if(CONFIG_IPC_MAJOR_3)
 	)
 elseif(CONFIG_IPC_MAJOR_4)
 	if(CONFIG_SAMPLE_SMART_AMP STREQUAL "m")
-		add_subdirectory(${SOF_SAMPLES_PATH}/audio/smart_amp_llext
-				 ${PROJECT_BINARY_DIR}/smart_amp_llext)
+		add_subdirectory(${SOF_SAMPLES_PATH}/audio/smart_amp_test_llext
+				 ${PROJECT_BINARY_DIR}/smart_amp_test_llext)
```

Comment:
why the rename? was this a mistake that should be corrected separated, something done in a previous patch?

Reply:
@plbossart previously this name wasn't a problem, since each module had its own supporting processing code, so names could be arbitrary. Now we need automation, so names must be unified as `${MODULE}_llext`